client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.html

```diff
@@ -8,89 +8,189 @@
-                            </li>
-                            <li class="oui-tile__item"
-                                data-ng-if="dedicatedCloud.ipRipeCount > 0">
-                                <div class="oui-tile__definition">
-                                    <strong class="d-block"
-                                            data-translate="dedicatedCloud_nb_bloc_IP_ripe">
-                                    </strong>
-                                    <span data-ng-bind="dedicatedCloud.ipRipeCount"></span>
-                                </div>
-                            </li>
-                            <li class="oui-tile__item"
-                                data-ng-if="dedicatedCloud.ipArinCount > 0">
-                                <div class="oui-tile__definition">
-                                    <strong class="d-block"
-                                            data-translate="dedicatedCloud_nb_bloc_IP_arin">
-                                    </strong>
-                                    <span data-ng-bind="dedicatedCloud.ipArinCount"></span>
-                                </div>
-                            </li>
-                            <li class="oui-tile__item"
-                                data-ng-if="dedicatedCloud.ipRipeCount === 0">
-                                <div class="oui-tile__definition">
-                                    <strong class="d-block"
-                                            data-translate="dedicatedCloud_nb_bloc_IP">
-                                    </strong>
-                                    <span data-translate="dedicatedCloud_nb_bloc_IP_no_data"></span>
-                                </div>
-                            </li>
-                        </ul>
+                        <div class="oui-tile__item">
+                            <dl class="oui-tile__definition">
+                                <dt class="oui-tile__term"
+                                    data-translate="dedicatedCloud_description"></dt>
+                                <dd class="oui-tile__description"
+                                    data-ng-bind="dedicatedCloud.description"></dd>
+                            </dl>
+                            <div class="oui-tile__actions">
+                                <oui-action-menu
+                                    aria-label="{{:: 'menu_action_label' | translate:{t0: ('dedicatedCloud_description' | translate)} }}"
+                                    data-compact
+                                    data-align="end">
+                                    <oui-action-menu-item
+                                        aria-label="{{:: 'common_change' | translate }}"
+                                        data-text="{{:: 'common_change' | translate }}"
+                                        on-click="setAction('name/dedicatedCloud-name',  { value: dedicatedCloud.description, path: 'dedicatedCloud.description'})">
```

Comment:
`data-`

Also, you should make sure you always use the same indentation between `{` and the text (sometimes you insert spaces, sometimes you don't)